pandas/core/ops.py

```diff
@@ -582,6 +751,35 @@ def _get_op_name(op, special):
 DataFrame.{reverse}
 """
 
+_flex_comp_doc_FRAME = """
+Flexible wrappers to comparison operators (specifically ``{name}``).
+
+Wrappers (``eq``, ``ne``, ``le``, ``lt``, ``ge``, ``gt``) are equivalent to
+operators (``==``, ``=!``, ``<=``, ``<``, ``>=``, ``>``) with support to choose
+axis (rows or columns) for comparison.
+
+Parameters
+----------
+other : DataFrame
+axis : {{0, 1, 'columns', 'rows'}}
+level : int or name
+    Broadcast across a level, matching Index values on the
+    passed MultiIndex level
+
+Returns
+-------
+result : DataFrame
+    Consisting of boolean values
```

Comment:
it's a bit redundant, but a bit more explanation, like "Result of the comparison.". Note the period at the end.